rector.yaml

```diff
@@ -0,0 +1,29 @@
+parameters:
+    autoload_paths:
+        - 'vendor/squizlabs/php_codesniffer/autoload.php'
+    paths:
+        - 'src'
+        - 'tests'
+        - 'bin'
+    exclude_paths:
+        - 'tests/*Fixtures/*'
+    auto_import_names: true
+    php_version_features: '7.4'
+    sets:
+        - 'action-injection-to-constructor-injection'
+        - 'array-str-functions-to-static-call'
+        - 'celebrity'
+        - 'doctrine'
+        - 'phpstan'
+        - 'phpunit-code-quality'
+        - 'solid'
+        - 'early-return'
+        - 'doctrine-code-quality'
+        - 'dead-code'
+        - 'code-quality'
+        - 'php71'
+        - 'php72'
+        - 'php73'
+        - 'php74'
```

Comment:
Also I recommend `privatization` set
https://www.tomasvotruba.com/blog/2020/03/30/dont-show-your-privates-to-public

Reply:
Oh nice, totally something for us, thanks 😄

Just right up @nunomaduro alley.